config.sample.json

```diff
@@ -0,0 +1,45 @@
+{
+    "coinCapUri": "https://api.coincap.io/v2/assets/",
+    "currencies": {
+        "BCH": {
+            "address": "",
+            "currencyName": "bitcoincash",
+            "coinCapCurrencyName": "bitcoin-cash"
```

Comment:
No need since the rates1 server uses the currency code

Reply:
Updated per comment. Removed `coinCapCurrencyName` property from each currency.